forte/base_classes/coupling_coefficients.h

```diff
@@ -0,0 +1,61 @@
+#ifndef COUPLINGCOEFFICIENTS_H
+#define COUPLINGCOEFFICIENTS_H
+
+#include <ambit/tensor.h>
+
+namespace forte {
+
+class CouplingCoefficients {
```

Comment:
I am wondering: what is the purpose of this class? Acting as container for coupling coefficients?
Base classes are important, so they should be documented like all the other ones in the same folder. E.g. see MOSpaceInfo or the ActiveSpace solver. Also I would suggest renaming to `CICouplingCoefficients` since this is specific to CI (e.g., it does not apply to methods like DMRG).

Reply:
Yes, it is a container for coupling coefficients, similar to the RDMs class. Unfortunately we need this right now for the pilot implementation. A more efficient implementation would be directly sigma build so that this class can be eliminated.

Reply:
@fevangelista I have changed the class name to CICouplingCoefficients